thoth/storages/graph/models.py

```diff
@@ -72,7 +72,6 @@ class PythonPackageVersion(Base, BaseExtension):
     )
     is_missing = Column(Boolean, nullable=False, default=False)
     provides_source_distro = Column(Boolean, nullable=False, default=True)
-
     # Relations
     dependencies = relationship("DependsOn", back_populates="version")
     solvers = relationship("Solved", back_populates="version")
```

Comment:
You will need to add a relation to `ImportPackage`. I think you will need an association table for N-N relation here (as one package might provide the same import as another one):

https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html#many-to-many

Reply:
resolved by 8a428279967604e0e603a2556370045264245cbd